src/containers/edm/propertytypes/PropertyTypeDetailsContainer.js

```diff
@@ -32,19 +38,113 @@ const DeleteButton = styled(StyledButton)`
   margin-top: 18px;
 `;
 
+const RadiosRowWrapper = styled.div`
+  display: flex;
+  flex-direction: row;
+  label {
+    margin-right: 20px;
+  }
+  input {
+    margin-right: 8px;
+  }
+`;
+
 /*
  * types
  */
 
 type Props = {
   actions :{
     localDeletePropertyType :RequestSequence;
+    localUpdatePropertyTypeMeta :RequestSequence;
+    resetRequestState :(actionType :string) => void;
   };
   entityTypes :List<Map<*, *>>;
   propertyType :Map<*, *>;
+  updateRequestState :RequestState;
 };
 
-class PropertyTypeDetailsContainer extends React.Component<Props> {
+type State = {
+  modalState :{
+    actionPrimary :() => void;
+    isVisible :boolean;
+    textTitle :string;
+    textContent :string;
+  };
+}
+
+const INITIAL_MODAL_STATE = {
+  actionPrimary: () => {},
+  isVisible: false,
+  textTitle: '',
+  textContent: '',
+};
+
+class PropertyTypeDetailsContainer extends Component<Props, State> {
+
+  constructor(props :Props) {
+
+    super(props);
+
+    this.state = {
+      modalState: INITIAL_MODAL_STATE,
+    };
+  }
+
+  componentDidUpdate(prevProps :Props) {
+
+    const { actions, updateRequestState } = this.props;
+    if (prevProps.updateRequestState === RequestStates.PENDING && updateRequestState === RequestStates.SUCCESS) {
+      this.resetModalState();
+      actions.resetRequestState(PropertyTypesActions.LOCAL_UPDATE_PROPERTY_TYPE_META);
+    }
+  }
+
+  resetModalState = () => {
+
+    this.setState({
+      modalState: INITIAL_MODAL_STATE,
+    });
+  }
+
+  handleOnChangePii = (event :SyntheticInputEvent<HTMLElement>) => {
+
+    const { actions, propertyType } = this.props;
+
+    let pii;
+    if (event.target.id === PII_YES_RADIO_ID) {
```

Comment:
Instead of `e.target.id`, consider using `e.target.checked` You can avoid this string equality business that way.

Reply:
No particular reason. There's no designs for the edm app, so I went with radios because I thought it was visually clearer to convey that this is a boolean value.

Reply:
![Screen Shot 2019-07-10 at 1 46 42 PM](https://user-images.githubusercontent.com/440299/61003604-2f9f7500-a319-11e9-87b9-37807c89a4a1.png)

At the moment, they layout has a header and then under it the value, and radios work well with that. If I were to use a checkbox, what would I specify for the checkbox's label?

Reply:
Does this not suffice?
![image](https://i.gyazo.com/fe09645b1fb93284d897fc2f870ca368.gif)